morituri/command/cd.py

```diff
@@ -134,10 +134,26 @@ def do(self):
         if not self.program.metadata:
             # fall back to FreeDB for lookup
             cddbid = self.ittoc.getCDDBValues()
-            cddbmd = self.program.getCDDB(cddbid)
-            if cddbmd:
-                sys.stdout.write('FreeDB identifies disc as %s\n' % cddbmd)
 
+            allow_freedb = getattr(self.options, 'allow_freedb', False)
+
+            sfdb = simplefreedb.SimpleFreeDB()
+            matches = sfdb.query(cddbid[0], cddbid[1], cddbid[2:-1], cddbid[-1])
+            if matches:
+                match = matches[0] # TODO: honor --prompt
```

Comment:
If you take a gander at the work I did to rewrite AccurateRip, I factored a generic prompt function you might find useful here.

Reply:
`git grep -i prompt` doesn't seem to raise anything related. Can you point out where I can find it?

Reply:
Oh, it was @ https://github.com/RecursiveForest/whipper/commit/ca33dcfe79af22e9ee3759659b5473bc517f0746#diff-aa28dbdd5d3fa03298c07270deac1ec1R40 , not part of the AR PR.

Reply:
Oh, but that's not mainline, or is it?